EIPS/eip-3670.md

```diff
@@ -87,19 +81,18 @@ Each case should be tested for creation transaction, `CREATE` and `CREATE2`.
 - EOF code with data section containing bytes that are undefined instructions
 - Legacy code containing undefined instruction
 - Legacy code ending with incomplete PUSH instruction
-- Legacy code ending with any valid non-terminating instruction
 
 ### Invalid codes
 
 - EOF code containing undefined instruction
 - EOF code ending with incomplete `PUSH` instruction
     - This can include `PUSH` instruction unreachable by execution, e.g. after `STOP`
-- EOF code ending with any defined instruction other than terminating ones
-    - This can include codes where non-terminating instruction is not reachable, e.g. `0030` (`STOP ADDRESS`).
 
 ## Reference Implementation
 
 ```python
+from eip3540 import ValidationException
```

Comment:
I omitted this import from other eips, I think it only takes space

Reply:
Done.